src/senaite/sync/browser/templates/sync.pt

```diff
@@ -23,196 +23,40 @@
-                        <input type="text"
-                               size="45"
-                               class="form-control"
-                               id="content_types"
-                               tal:attributes="value view/content_types|nothing"
-                               name="content_types"/>
-                      </div>
-                    </div>
-                  </li>
-                </ul>
-              </div>
-            </div>
-          </div>
-        </div>
-
-        <input class="btn btn-success btn-sm allowMultiSubmit"
-               type="submit"
-               name="fetch"
-               i18n:attributes="value"
-               value="Fetch"/>
-
-      </form>
+      <form action="sync_add">
+        <input type="submit" value="Add New Remote" />
+      </form><br><br>
 
       <!-- Storage View -->
       <tal:storage repeat="storage view/storage">
+        <div style="border:double; padding:10px">
         <form
           name="data_form"
           method="POST">
 
           <dl class="collapsible">
 
             <dt class="collapsibleHeader">
-              Data for <span tal:replace="storage"/>
+              <span tal:replace="python: storage.upper()"/>
               (<span tal:replace="python:len(view.storage[storage]['ordered_uids'])"/> Items)
             </dt>
             <input type="hidden" name="dataform" value="1"/>
             <input type="hidden" name="domain_name" value="" tal:attributes="value storage" />
 
             <dd class="collapsibleContent">
-              <div>
-                  Import settings: <span tal:replace="python:view.storage[storage]['configuration'].get('import_settings')"/>
-                  Import registry: <span tal:replace="python:view.storage[storage]['configuration'].get('import_registry')"/>
-                  Import users: <span tal:replace="python:view.storage[storage]['configuration'].get('import_users')"/>
+              <div
+                      tal:define="content_types python:view.storage[storage]['configuration'].get('content_types');
+                                  unwanted_content_types python:view.storage[storage]['configuration'].get('unwanted_content_types');
+                                  prefixable_types python:view.storage[storage]['configuration'].get('prefixable_types');"
+              >
+                  <b>Prefix: <span style="font-size: medium" tal:replace="python: view.storage[storage]['configuration'].get('prefix', 'Not defined')"/></b><br><br>
+                  Settings will <span tal:condition="python: not view.storage[storage]['configuration'].get('import_settings')">NOT</span>,
+                  Registry will <span tal:condition="python: not view.storage[storage]['configuration'].get('import_registry')">NOT</span>,
+                  Users will <span tal:condition="python: not view.storage[storage]['configuration'].get('import_users')">NOT</span> be imported.<br><br>
```

Comment:
Would be nice to create a function (something like `def get_storage_setting(self, parameter, section='configuration')` in the view to make the retrieval of all these settings easier. A thumb of rule is to always try to keep the tpl as much simpler as possible, with less logic as possible.